### ISSUE TITLE
modifications 

### ISSUE THREAD

**Issue by im420Mikey420** [BearDooks/Ender3Pro_Firmware#1]
i dont know if you will even see this im not an expert on github but i was wondering if you could make some size adjustments on this firmware for me as i have the ender extender on my ender 3 pro with the sprite hot end and bl touch

**Comment by BearDooks** [comment]
Do you also have the BTT SKR Mini e3 v3.0 board installed? This firmware is specifically for that board.

Chuck

**Comment by im420Mikey420** [comment]
Yes I do  it’s just my ender is not stock anymore it now has a 400x400x500 print area
Sent from my iPhone

On Jul 27, 2024, at 6:47 PM, Chuck ***@***.***> wrote:

﻿

Do you also have the BTT SKR Mini e3 v3.0 board installed? This firmware is specifically for that board.

Chuck

**Comment by BearDooks** [comment]
OK. I am at work now, but I will try to get to this tonight and see if I can make something for you.

Chuck

**Comment by im420Mikey420** [comment]
Ok thank you so Much
Sent from my iPhone

On Jul 29, 2024, at 11:15 AM, Chuck ***@***.***> wrote:

﻿

OK. I am at work now, but I will try to get to this tonight and see if I can make something for you.

Chuck

**Comment by BearDooks** [comment]
OK. I made some changes and compiled the new firmware. You can find it here

https://github.com/BearDooks/Ender3Pro_Firmware/tree/main/Precompiled%20Firmware/firmware_with_extender

or here

https://github.com/BearDooks/Ender3Pro_Firmware/releases/tag/v2.0e

All I did was change the X and Y limits to 400 and the Z limit to 500, which should give you the 400x400x500 print bed size. Please note that since I don't have that extender, or even this printer anymore, I can't test this firmware, so you need to use it at your own risk.

Chuck

**Comment by im420Mikey420** [comment]
If it’s same firmware just with the print size change it should be ok I would think
Sent from my iPhone

On Jul 29, 2024, at 7:23 PM, Chuck ***@***.***> wrote:

﻿

OK. I made some changes and compiled the new firmware. You can find it here

https://github.com/BearDooks/Ender3Pro_Firmware/tree/main/Precompiled%20Firmware/firmware_with_extender

or here

https://github.com/BearDooks/Ender3Pro_Firmware/releases/tag/v2.0e

All I did was change the X and Y limits to 400 and the Z limit to 500, which should give you the 400x400x500 print bed size. Please note that since I don't have that extender, or even this printer anymore, I can't test this firmware, so you need to use it at your own risk.

Chuck

**Comment by BearDooks** [comment]
It should be. Just let me know if it works 

Chuck

**Comment by im420Mikey420** [comment]
Will do I’ll pop it in tomorrow morning
Sent from my iPhone

On Jul 29, 2024, at 7:58 PM, Chuck ***@***.***> wrote:

﻿

It should be. Just let me know if it works

Chuck

**Comment by im420Mikey420** [comment]
It loaded just fine and everything worked but when I was heating up the bed to do the offset and to lvl it it restarted itself after it hit 70+
Sent from my iPhone

On Jul 29, 2024, at 9:46 PM, Michael Schindler ***@***.***> wrote:

﻿ Will do I’ll pop it in tomorrow morning
Sent from my iPhone

On Jul 29, 2024, at 7:58 PM, Chuck ***@***.***> wrote:

﻿

It should be. Just let me know if it works

Chuck

**Comment by BearDooks** [comment]
I'm sorry, I don't understand. The printer restarted when the bed temp went above 70?

**Comment by im420Mikey420** [comment]
Yes but with previous firmware I had installed back on it heats up just fine
Sent from my iPhone

On Jul 31, 2024, at 9:28 AM, Chuck ***@***.***> wrote:

﻿

I'm sorry, I don't understand. The printer restarted when the bed temp went above 70?

**Comment by BearDooks** [comment]
I just checked the config and the max bed temp is set to 125 with an overshoot of 10, so i don't think that could be the issue. Online a few people said to make sure the power switch on the PSU is set to the correct voltage. As I am unsure where you live, in the US some people had noticed it was set for 230 rather than 115 and it would short out

**Comment by im420Mikey420** [comment]
It’s set right Iv been using the printer for 3 years now I’ll try again and see if it was just a fluke thing
Sent from my iPhone

On Jul 31, 2024, at 10:12 AM, Chuck ***@***.***> wrote:

﻿

I just checked the config and the max bed temp is set to 125 with an overshoot of 10, so i don't think that could be the issue. Online a few people said to make sure the power switch on the PSU is set to the correct voltage. As I am unsure where you live, in the US some people had noticed it was set for 230 rather than 115 and it would short out

**Comment by BearDooks** [comment]
Just checking in to see if this is still an issue?
